refinery/data_set_manager/tests.py

```diff
@@ -1,18 +1,20 @@
 import json
-from StringIO import StringIO
-
 import re
+from StringIO import StringIO
 
 from django.contrib.auth.models import User
 from django.core.files.uploadedfile import (InMemoryUploadedFile,
                                             SimpleUploadedFile)
-
 from django.http import QueryDict
 from django.test import TestCase
-
 from rest_framework.test import APIClient, APIRequestFactory, APITestCase
 
-from .models import Assay, AttributeOrder, Study, Investigation, Node
+from core.models import DataSet, ExtendedGroup, InvestigationLink
```

Comment:
isort wants to put relative imports at the end, which doesn't seem bad to me, but I'll look into overriding that: #1830.